### DISCUSSION TITLE
Lottie Texture Player initial state

### DISCUSSION THREAD

**Discussion by tgd-git** [cables-gl/cables_docs#266] [! Welcome - readme first!]
I have noticed two things with the lottie player op when freshly opening a patch.
1. It wont show a texture until you change the frame number.
2. It ignores the filter setting (seems only when set to linear, mipmap works). Always opens with the default (nearest), although showing otherwise in the config dropdown.

**Comment by tgd-git** [comment]
More (not related to init state):
3. The texture has black borders on soft edges of transparency unless you use BlendMode op and check Premultiplied. 
Drawback then is, you can not fade out via the material alpha.
4. Luma Masks dont work (the mask is rendered, does not mask). They work fine with the LottieSVGPlayer.
5. Blur does not work. Works fine with SVGPlayer.

**Comment by tgd-git** [comment]
6. play checkbox does not do anything.
7. rewind button does not do anything, although it might depend on play (see 6.)

also forget 4. and 5. i did zero research ;) https://airbnb.io/lottie/#/supported-features

**Comment by pandrr** [comment]
hi,

there is now Ops.Dev.LottieTexturePlayer2 on dev, which should fix all those things.
it has the same timing switch interface like lottieSvg op.

let me know if there are issues left

**Comment by tgd-git** [comment]
linear filter is still ignored

**Comment by tgd-git** [comment]
When using blend mode and premultiplied checkbox, how to fade out? material alpha wont work. is there any other way?
![image](https://user-images.githubusercontent.com/2562637/173382682-1ae5672a-896a-47cf-8356-83ab7b7c08cf.png)
